esbuild.cronjobs.mjs

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { build } from "esbuild";
+
+// Theoretically, since the cron jobs run in Node, we don't need a bundler,
+// since there are no HTTP requests we're trying to optimise. However, plain
+// Node.js can't resolve module imports without file extensions (e.g.
+// `"../db/tables/subscribers"`), so all import specifiers in *any* module that
+// gets loaded by the cron jobs would need to use file extensions, which should
+// also match those of the generated files (i.e. .js rather than .ts). Some of
+// those modules are also used in our website, where this is bound to cause its
+// own issues in Next.js. Thus, esbuild is a compromise that can resolve these
+// import specifiers for us.
+build({
+  entryPoints: ["./src/cronjobs/**/*.tsx"],
+  tsconfig: "tsconfig.cronjobs.json",
+  bundle: true,
+  platform: "node",
+  format: "esm",
+  outdir: "dist/cronjobs",
+  sourcemap: true,
+  target: "node20.12",
```

Comment:
I think it can be just `node` or `node20` if you want to be more specific - do you think it's worth specifying it here? The environment running the cron job (k8s `CronJob` using the container we build in `Dockerfile`) will be what's actually available in the environment.

Reply:
What I like about saying `20.12` is that it will show up on a project-wide search for that number, so that we won't forget to update it when updating the other instances. Using just `20` wouldn't be a problem for the coming future, but makes it easy to miss when we upgrade to Node 22.

As for the document/wiki/README, I'd actually like to write a small script that just checks all instances and errors out if they're not aligned, that we can run in CI.

Reply:
https://github.com/mozilla/blurts-server/pull/4404